### ISSUE TITLE
Fallback GTK theme on Manjaro KDE

### ISSUE THREAD

**Issue by unfa** [tenacityteam/tenacity-legacy#576]
### Guidelines

- [X] I have read the guidelines.

### Version/Commit hash

657a83b9efa53e66d0273de992ad2944fa585dab

### Describe the bug.

Tenacity seems to be using the old Redmond theme, instead of what every other GTK application on my system uses.
![image](https://user-images.githubusercontent.com/491704/132026513-222437e8-2259-42e8-8efd-38da300af945.png)

I did not have this problem with Audacity.

### Expected behavior

I'd expect Tenacity to use a system-wide GTK theme and look decent :D

### OS

Manjaro KDE

### This issue is not a duplicate

- [X] I have confirmed this issue isn't a duplicate.

**Comment by n0toose** [comment]
Hi, sorry to ask like that, but why a commit from July 31st? Is there some sort of package that we may not be aware of?

**Comment by unfa** [comment]
I've used this AUR PKGBUILD to install Tenacity on Manjaro as it seemed to be the easiest way - it built it from source.
https://aur.archlinux.org/packages/tenacity-git

BTW - I should update and see what happens.

**Comment by unfa** [comment]
I've updated and now it doesn't start complaining about a missing library:
```
$ tenacity 
tenacity: error while loading shared libraries: libwx_baseu-3.1.so.4: cannot open shared object file: No such file or directory
```

**Comment by unfa** [comment]
I probably need to install some wxwidgets stuff, but I have no idea what exactly.
Could anybody figure out a dependancy list fro Arch Linux / Manjaro Linux?

Here's packages listed under wxwidgets on my Manjaro KDE system:

![image](https://user-images.githubusercontent.com/491704/132073050-25036565-dbeb-49b8-b44c-9bf60bc4f29c.png)

**Comment by Be-ing** [comment]
It's not your fault at all for using the package, but we've specifically asked packagers not to package Tenacity before we make a release. I took a quick look at the [PKGBUILD](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=tenacity-git) and the cause of the problem isn't obvious to me (I also don't use Arch or Manjaro). I recommend building from source using the [instructions](https://github.com/tenacityteam/tenacity/blob/master/BUILDING.md#arch) which have been confirmed to work by multiple Arch users.

**Comment by unfa** [comment]
I've followed the linked instructions for Arch Linux on Manjaro, but was unable to get Tenacity to start up.
I'm still missing the listed library file.

I'll go back to Audacity for now. Manjaro's packaged version is pre-MuseGroup, so it has no telemetry.
I'll wait for an official Tenacity package for Manjaro.

**Comment by Be-ing** [comment]
You're not the first user to run into this: #554.

I am not able to reproduce it on Fedora. Can you try running Tenacity from the CMake build directory?

```
cd wherever-you-cloned-tenacity
build/bin/Debug/tenacity
```

**Comment by Eylok** [comment]
There's a pinned comment in the tenacity-git aur package page that explains why it fails to start. https://aur.archlinux.org/packages/tenacity-git
It's using 'wxgtk3-dev-opt', which installs to /opt/wxgtk-dev/lib/, and tenacity can't find the library there.
There's a few options on how to fix it in the comments, like exporting LD_LIBRARY_PATH with the path that contains the libraries.

Also the PKGBUILD includes portsmf as a dependency, which for me (and others in the comments i think) causes the compilation to fail, so if you are building it maybe it's best to uninstall that package.
With a modified PKGBUILD it compiles and runs on arch. I changed 'wxgtk3-dev-opt' for 'wxgtk3-dev-light', removed portsmf, and changed the wx-config and libraries locations in the build section. With those modifications it works on my computer.
Here's a gist with the modifications that i made, maybe it'll be useful:
https://gist.github.com/Eylok/53a9dc4fd7619a77447fa20c2a1eebf6

**Comment by Be-ing** [comment]
Thanks. If you could work with whoever made that broken tenacity-git package to put your changes in that package, that would save us quite a bit of trouble upstream.

**Comment by Eylok** [comment]
Honestly i don't have experience with how the aur package system works, or at least how to do in case i want to suggest changes, etc. I'm already reading the wiki, and trying to familiarize myself with the process. No promises, but i'll see what i can do.
Maybe in the meantime it's a good idea to add some info to BUILDING.md, explaining the aur package situation.

**Comment by RiedleroD** [comment]
I didn't have the theme problem before, but I did get the compilation error - according to the tenacity account on mastodon, the latter should be fixed - will test this out later today.

**Comment by n0toose** [comment]
> I didn't have the theme problem before, but I did get the compilation error - according to the tenacity account on mastodon, the latter should be fixed - will test this out later today.

Yeah, as said on the Mastodon account itself, it does work now. We and our followers confirmed that, it's just that we changed way too many things in the building process and the maintainers of the package took some time to catch-up. What's weird about this issue is that it seems to be Manjaro-specific, we may have to reach out to the Manjaro community to get some help diagnosing the issue.

**Comment by RiedleroD** [comment]
still doesn't compile on my end - some problem with memcpy not being included properly as it seems. Should I open a separate issue for this?

**Comment by n0toose** [comment]
Sorry for the late response, I'm really not sure whether this is a Manjaro issue, honestly. Does it still happen with upstream Audacity?

**Comment by User-green** [comment]
I can perform testing on Manjaro with KDE, is it still needed?
